Model/Sync/Catalog/Processor.php

```diff
@@ -844,4 +848,22 @@ private function updateSyncTable($syncDataRecord)
             [$syncDataRecord]
         );
     }
+
+    private function handleRequest($itemEntityId, $yotpoItemData, $apiRequestParams) {
+        $syncMethod = $apiRequestParams['method'];
+        $yotpoItemId = $apiRequestParams['yotpo_id'];
+        $yotpoParentItemId = $apiRequestParams['yotpo_id_parent'];
+
+        switch ($syncMethod) {
+            case 'createProduct':
+            case 'updateProduct':
+                return $this->handleProductUpsert($itemEntityId, $yotpoItemData, $yotpoItemId);
+            case 'createProductVariant':
+            case 'updateProductVariant':
+                return $this->handleVariantUpsert($itemEntityId, $yotpoItemData, $yotpoParentItemId, $yotpoItemId);
+            default:
+                $response = $this->processRequest($apiRequestParams, $yotpoItemData);
```

Comment:
`processRequest` should not include references or usages for the 4 cases above, remove the relevant code from there.